setup.py

```diff
@@ -31,6 +33,99 @@
         version = match.group(1)
 
 
+def _get_help_output(topic, lines=False):
+    TRANSLATE = {
+        'tar-content': 'tar_content',
+    }
```

Comment:
Right now it's the only one and I thought I'd rather make this explicit than just changing anything which comes up with a dash.